dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/bnd.bnd

```diff
@@ -116,7 +123,11 @@ instrument.classesExcludes: \
   org/eclipse/persistence/internal/helper/*.class
 
 -buildpath: \
-    org.eclipse.persistence.core;version=3.0.0,\
-    org.eclipse.persistence.jpa;version=3.0.0,\
+    org.eclipse.persistence:org.eclipse.persistence.asm;version=${eclFullVersion},\
+    org.eclipse.persistence:org.eclipse.persistence.corba;version=${eclFullVersion},\
+    org.eclipse.persistence:org.eclipse.persistence.core;version=${eclFullVersion},\
+    org.eclipse.persistence:org.eclipse.persistence.jpa;version=${eclFullVersion},\
+    org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;version=${eclFullVersion},\
+    org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen;version=${eclFullVersion},\
```

Comment:
Should these have strategy=exact and reference the -R1.5 or -M1 like mentioned earlier.